packages/sdk-core/src/model/http/SubmissionUrlInformation.ts

```diff
@@ -30,6 +30,22 @@ export class SubmissionUrlInformation {
         return result.href;
     }
 
+    /**
+     * Converts full submission JSON URL to PlCrashReporter submission URL
+     * @param submissionUrl Backtrace Submission URL
+     */
+    public static toPlCrashReporterSubmissionUrl(submissionUrl: string): string {
+        return this.changeSubmissionFormat(submissionUrl, 'minidump');
+    }
```

Comment:
it's a little bit weird to include this function here. It's highly vendor specific, and will be used only in React Native. As it is identical to `toMinidumpSubmissionUrl`, why not just keep that one?

Reply:
your suggestion is to keep only `changeSubmissionFormat` function are remove plcrashreporter and minidump functions?

Reply:
Either that or keep `toMinidumpSubmissionUrl`.

Reply:
ah ok - I see the issue. this is wrong - plcrashreporter submission URL should be plrash instead of minidump. Modified it to the desired format